iron/distribution.yaml

```diff
@@ -8363,16 +8363,16 @@ repositories:
     doc:
       type: git
       url: https://github.com/ros-industrial/ur_msgs.git
-      version: foxy
+      version: humble
```

Comment:
The branch for the `doc` section and the `source` section are different here.  While it is allowed, it is a bit unusual.  Is that what you meant to do?

Reply:
Yes, but looking back at noetic, I probably should have done it the other way around.

Reply:
That's fine, I just wanted to double-check.